src/flint/flint_base/flint_base.pxd

```diff
@@ -23,6 +23,7 @@ cdef class flint_mpoly(flint_elem):
     cdef _mul_mpoly_(self, other)
 
     cdef _divmod_mpoly_(self, other)
+    cdef _truediv_scalar_(self, other, assume_exact: bool)
```

Comment:
I would generally just make this two functions rather than having a bool flag so we have:
```
cdef _truediv_scalar_(self, other)
cdef _truediv_exact_scalar_(self, other)
```